lib/Doctrine/deploy/sampleData/ServiceEndpoints.xml

```diff
@@ -285,11 +285,385 @@
+        <DESCRIPTION>site LFC</DESCRIPTION>
+        <SERVICE_TYPE>Local-LFC</SERVICE_TYPE>
+        <HOST_IP>185.34.482.95</HOST_IP>
+        <SCOPE>Local</SCOPE>
+        <CORE></CORE>
+        <IN_PRODUCTION>Y</IN_PRODUCTION>
+        <NODE_MONITORED>Y</NODE_MONITORED>
+        <SITENAME>SSC</SITENAME>
+        <COUNTRY_NAME>Slovenia</COUNTRY_NAME>
+        <COUNTRY_CODE>SI</COUNTRY_CODE>
+        <ROC_NAME>NGI_SI</ROC_NAME>
+        <URL></URL>
+    </SERVICE_ENDPOINT>
+    <SERVICE_ENDPOINT PRIMARY_KEY="73221G0">
+        <PRIMARY_KEY>73221G0</PRIMARY_KEY>
+        <HOSTNAME>srm.ssc.si</HOSTNAME>
+        <GOCDB_PORTAL_URL>https://testing.host.com/portal/index.php?Page_Type=View_Object&amp;object_id=73221&amp;grid_id=0</GOCDB_PORTAL_URL>
+        <DESCRIPTION>srm</DESCRIPTION>
+        <SERVICE_TYPE>SRM</SERVICE_TYPE>
+        <HOST_IP>185.34.482.45</HOST_IP>
+        <SCOPE>EGI</SCOPE>
+        <CORE></CORE>
+        <IN_PRODUCTION>Y</IN_PRODUCTION>
+        <NODE_MONITORED>Y</NODE_MONITORED>
+        <SITENAME>SSC</SITENAME>
+        <COUNTRY_NAME>Slovenia</COUNTRY_NAME>
+        <COUNTRY_CODE>SI</COUNTRY_CODE>
+        <ROC_NAME>NGI_SI</ROC_NAME>
+        <URL></URL>
+    </SERVICE_ENDPOINT>
+    <SERVICE_ENDPOINT PRIMARY_KEY="9980G0">
+        <PRIMARY_KEY>9980G0</PRIMARY_KEY>
+        <HOSTNAME>bdii4.izolamrf.si</HOSTNAME>
+        <GOCDB_PORTAL_URL>https://testing.host.com/portal/index.php?Page_Type=View_Object&amp;object_id=9980&amp;grid_id=0</GOCDB_PORTAL_URL>
+        <DESCRIPTION>site BDII</DESCRIPTION>
+        <SERVICE_TYPE>Site-BDII</SERVICE_TYPE>
+        <HOST_IP>191.640.12.45</HOST_IP>
+        <SCOPE>EGI</SCOPE>
+        <CORE></CORE>
+        <IN_PRODUCTION>N</IN_PRODUCTION>
+        <NODE_MONITORED>Y</NODE_MONITORED>
+        <SITENAME>Izola MRF</SITENAME>
+        <COUNTRY_NAME>Slovenia</COUNTRY_NAME>
+        <COUNTRY_CODE>SI</COUNTRY_CODE>
+        <ROC_NAME>NGI_SI</ROC_NAME>
+        <URL></URL>
+    </SERVICE_ENDPOINT>
+    <SERVICE_ENDPOINT PRIMARY_KEY="9933G0">
+        <PRIMARY_KEY>9933G0</PRIMARY_KEY>
+        <HOSTNAME>ce01.izolamrf.si</HOSTNAME>
+        <GOCDB_PORTAL_URL>https://testing.host.com/portal/index.php?Page_Type=View_Object&amp;object_id=9933&amp;grid_id=0</GOCDB_PORTAL_URL>
+        <DESCRIPTION>arc-ce</DESCRIPTION>
+        <SERVICE_TYPE>ARC-CE</SERVICE_TYPE>
+        <HOST_IP>190.884.01.90</HOST_IP>
+        <SCOPE>EGI</SCOPE>
+        <CORE></CORE>
+        <IN_PRODUCTION>N</IN_PRODUCTION>
+        <NODE_MONITORED>Y</NODE_MONITORED>
+        <SITENAME>Izola MRF</SITENAME>
+        <COUNTRY_NAME>Slovenia</COUNTRY_NAME>
+        <COUNTRY_CODE>SI</COUNTRY_CODE>
+        <ROC_NAME>NGI_SI</ROC_NAME>
+        <URL></URL>
+    </SERVICE_ENDPOINT>
+    <SERVICE_ENDPOINT PRIMARY_KEY="10007G0">
+        <PRIMARY_KEY>10007G0</PRIMARY_KEY>
+        <HOSTNAME>uni-bdii.uni-lj.si</HOSTNAME>
```

Comment:
looks like `uni-lj.si` is a thing, could this be changed?

Reply:
and changed in other places as well.

Reply:
yep will change this sitename and related services too